frontend/src/profile-registration/EmailInput.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export class EmailInput extends Component {
+    render() {
+        const { onChange } = this.props
+        const id = uuidv4()
+        return (
+            <fieldset>
+                <input
+                    id={id}
+                    type="email"
+                    name="email"
+                    required
+                    onChange={e => onChange(e.currentTarget.value)} />
```

Comment:
provide value for input 
https://reactjs.org/docs/forms.html#controlled-components